api/helpers.go

```diff
@@ -22,3 +25,17 @@ func WriteBytes(w http.ResponseWriter, status int, text []byte) {
 	w.WriteHeader(status)
 	w.Write(text)
 }
+
+// WriteError to the response with message and log error message
+func WriteError(w http.ResponseWriter, status int, errorMsg string, err error) {
+	logger.Log.Errorf("error %s, %s", errorMsg, err.Error())
+	errMsg := fmt.Sprintf(`{"success":false, "message":"%s", "reason": "%s"}`, errorMsg, err.Error())
+	Write(w, status, errMsg)
+}
+
+// WriteSuccessWithJSON sends response with statusOK to request and log success msg
+func WriteSuccessWithJSON(w http.ResponseWriter, status int, res []byte, msg string) {
+	retMsg := fmt.Sprintf(`{"success" : true, "response":%s}`, res)
+	logger.Log.Info(msg)
+	Write(w, http.StatusOK, retMsg)
```

Comment:
`Write(w, status, retMsg)` or if success is always StatusOK you can remove status from the arguments